templates/openwisp2/urls.py

```diff
@@ -16,6 +16,11 @@
     {% if openwisp2_network_topology %}
     url(r'^', include('openwisp_network_topology.urls')),
     {% endif %}
+    {% if openwisp2_radius %}
+        url(r'^', include('openwisp_radius.urls')),
+        url(r'^api/v1/', include('openwisp_utils.api.urls')),
+        url(r'^api/v1/', include('openwisp_users.api.urls')),
```

Comment:
I think these two should be added independently from this if and should be always added, but I'm not sure if it's being done in other PRs

Reply:
Makes sense! :smile: